go/oasis-test-runner/oasis/runtime.go

```diff
@@ -274,7 +274,7 @@ func (rt *Runtime) toRuntimeBundle(deploymentIndex int) (*bundle.Bundle, error)
 		if err != nil {
 			return nil, fmt.Errorf("oasis/runtime: failed to read ELF binary: %w", err)
 		}
-		_ = bnd.Add(elfBin, binBuf)
+		_ = bnd.Add(elfBin, bundle.NewBytesData(binBuf))
```

Comment:
Currently we are only adding files, so generalization was not needed (and sending file would make the comment of `Add` more clear). However, since you want to generalize, this is fine. Resolving.

Reply:
There are also external uses of this API which use byte arrays (e.g. when computing and adding the signature).

Reply:
Oh, I missed that. The current implementation makes a lot more sense now.